src/types.ts

```diff
@@ -244,6 +244,11 @@ export interface ViewState {
   visibleWidth?: number;
   rowsPerPage?: number;
   page?: number;
+  isMultiPage?: boolean;
+  maxLineCount?: number;
+  tableType?: number;
```

Comment:
Is `tableType` something temporary or why is that needed? Is the `usePagination` property on the layout not enough?

Reply:
The usePagination property on the layout not enough! Let say we have a pagination table but to estimate the number of rows in a pdf page we need to render it as a virtualized table (we use first data rows to render, so we don't need to fetch data since the data already exist). However, to render multi-page pdf, especially from the second page we have to use pagination table to do the job. Currently, using virtualized table to does not work for other pages than the first page.

Reply:
I just replaced tableType to usePagination to make it consistent with layout.usePagination.